mobile/src/bundles/employer/components/employer-onboarding-form/employer-onboarding-form.tsx

```diff
@@ -94,14 +94,11 @@ const EmployerOnboardingForm: React.FC<Properties> = ({
                             }}
                         />
                     </FormField>
-                    <Text category={TextCategory.H6}>Profile photo</Text>
                 </View>
                 <View>
                     <FormField
-                        errorMessage={
-                            errors.companyLogo && 'Company logo is required'
-                        }
```

Comment:
we shouldn't show validation error messages in the form?

Reply:
As I understood from communication with QA, a company logo is not required, but profile photo is required